### PR TITLE
3.9 - Backported Command::defaultName()

### PR DESCRIPTION
**Pull request by othercorey**
https://github.com/cakephp/cakephp/pull/13339

### CODE REVIEW

**Comment by othercorey** [comment]
Let's get this done.

**Comment by dereuromark** [comment]
I ticked off the box @ https://github.com/cakephp/cakephp/issues/13493 for this here

File: src/Console/Command.php [hunk 1]
```diff
@@ -102,6 +103,25 @@ public function getName()
...
+     * Returns the command name based on class name.
+     * For e.g. for a command with class name `UpdateTableCommand` the default
+     * name returned would be `'update_table'`.
+     *
+     * @return string
+     */
+    public static function defaultName()
+    {
+        $pos = strrpos(static::class, '\\');
+        /** @psalm-suppress PossiblyFalseOperand */
```
Comment (saeideng) [inline src/Console/Command.php:118]: I dont think we use the `psalm` in 3.x

**Review by saeideng — COMMENTED** [review]
LGTM

File: src/Console/Command.php (hunk 1 above)
Comment (dereuromark) [reply src/Console/Command.php:118]: but wouldnt be a big deal to keep, either.

Comment (saeideng) [reply src/Console/Command.php:118]: yes

**Pull request merged**